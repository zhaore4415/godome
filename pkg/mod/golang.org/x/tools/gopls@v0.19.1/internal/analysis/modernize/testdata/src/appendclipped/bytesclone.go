package appendclipped

import (
	"bytes"
)

var _ bytes.Buffer

func _(b []byte) {
	print(append([]byte{}, b...)) // want "Replace append with bytes.Clone"
}
