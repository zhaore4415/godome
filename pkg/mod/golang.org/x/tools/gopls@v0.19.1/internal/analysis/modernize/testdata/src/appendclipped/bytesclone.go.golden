package appendclipped

import (
	"bytes"
)

var _ bytes.Buffer

func _(b []byte) {
	print(bytes.Clone(b)) // want "Replace append with bytes.Clone"
}
